packages/runtime/container-runtime/src/opLifecycle/definitions.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IBatchMessage } from "@fluidframework/container-definitions";
+import { CompressionAlgorithms, ContainerRuntimeMessage, ICompressionRuntimeOptions } from "..";
+
+export interface IOutboxOptions {
+    readonly compressionOptions?: ICompressionRuntimeOptions;
+    readonly enableOpReentryCheck?: boolean;
+    readonly maxBatchSizeInBytes: number;
```

Comment:
Can you please add comments on what each property tracks? For example, I assume this is size on the wire, i.e. after (optional) compression, right?